### ISSUE TITLE
New Envato API 2015

### ISSUE THREAD

**Issue by themeblvd** [envato/envato-wordpress-toolkit#71]
Are there any plans to eventually get things moving towards the new Envato API?

http://marketblog.envato.com/releases/new-envato-api/

Just curious! Thanks for all you've done with this plugin and continued work on it.

**Comment by manojhl** [comment]
:+1:

**Comment by valendesigns** [comment]
Yes. The plugin is getting a complete rebuild. Which will begin in the next week or so. Cheers!
